source/direct_debit/index.html.md.erb

```diff
@@ -322,8 +341,4 @@ paying users receive their mandate reference number.
 
 ## Refunds
 
-[Contact us](/support_contact_and_more_information/#contact-us) if you need to issue refunds for any kind of Direct Debit payment.
-
-## Indemnity claims
-
-[Contact us](/support_contact_and_more_information/#contact-us) if you need to handle indemnity claims for any kind of Direct Debit payment.
+If a payment's status is `indemnityclaim` or a user asks you for a refund, you can process the refund by signing into your GoCardless account through the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/).
```

Comment:
If an indemnity claim has been made, the service doesn’t need to issue a refund: the paying user’s bank will do it.

Reply:
Thanks Alex - fixed in https://github.com/alphagov/pay-tech-docs/pull/306/commits/5aa961bb7886a311a3d22f9bd003a8c1fe598468